src/Functions/FunctionsHashing.h

```diff
@@ -71,8 +71,11 @@ namespace ErrorCodes
     extern const int SUPPORT_IS_DISABLED;
 }
 
+
 namespace impl
 {
+    constexpr auto EMPTY_HASH{0xe28dbde7fe22e41c};
```

Comment:
I think it's better to have different values for empty vs null. A table with no columns is different from a table with one column filled with NULLs.

(Idk if tables with no columns are allowed in CH, but still. E.g. maybe someone builds a query programmatically as "cityHash64($(list_of_columns))".)